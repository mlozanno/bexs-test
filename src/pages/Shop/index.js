import React from 'react';

const Shop = () => <div data-testid="shop-page" />;

export default Shop;
