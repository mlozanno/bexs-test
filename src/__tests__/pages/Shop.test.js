import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Shop from '~/pages/Shop';

describe('<Shop /> page', () => {
	it('should be a function', () => {
		expect(typeof Shop).toBe('function');
	});

	it('should be render component', () => {
		const { getByTestId } = render(<Shop />);

		const shop = getByTestId('shop-page');

		expect(shop).toMatchSnapshot();
	});
});
