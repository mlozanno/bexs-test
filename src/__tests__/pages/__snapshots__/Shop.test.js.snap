// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Shop /> page should be render component 1`] = `
<div
  data-testid="shop-page"
/>
`;
